### DISCUSSION TITLE
Custom keyboard shortcuts for everything

### DISCUSSION THREAD

**Discussion by IromTypo** [runelite/runelite#16722] [Ideas]
Wouldn't it be cool if we could manually add keyboard shortcuts to buttons? You would shift->right click a button, "set shortcut" and type in the letter or number you would want to use. Would be a great addition to Key Remapping!

**Comment by Nightfirecat** [comment]
In most cases, this amounts to macroing, so we cannot implement it.

**Reply by IromTypo** [reply]
Much sad :(
